src/main/scala/com/mwz/sonar/scala/scalastyle/ScalastyleSensor.scala

```diff
@@ -21,8 +21,8 @@ package scalastyle
 import java.io.File
 import java.nio.file.Paths
 
-import scala.collection.JavaConverters._
 import scala.collection.immutable.Seq
```

Comment:
> BTW, I remember we discussed this some time ago, but I would like to re-mention that the use of `Seq` is _(IMHO)_ discouraged, and the use of more concrete collections should be preffered. see [this](https://gist.github.com/djspiewak/2ae2570c8856037a7738#generic-interfaces) for a more detailed explanation.

yeah, I agree - I don't think we access any of the sequences by index, so it would definitely be good to make them more specific and use `List` whenever possible, or a [`Chain`](https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/data/Chain.scala) which is much more performant than `List`. Probably out of the scope of this PR though.